src/VirtualClient/VirtualClient.Actions.UnitTests/ApacheBench/ApacheBenchExecutorTests.cs

```diff
@@ -0,0 +1,328 @@
+﻿namespace VirtualClient.Actions.ApacheBench
```

Comment:
Can you please also add Functional Tests. It tests overall interaction right from the profile level.